packages/@glimmerx/storybook/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
We should combine this one with the root tsconfig.

Reply:
done. it is extending the base config now